kdestandardize.ado

```diff
@@ -1414,49 +1423,247 @@ prog def kdestandardize
+	qui: rename other_cnt othcnt
+	qui: destring othcnt, replace ignore("*,-R %")
+	la var othcnt "Other Student Involvemnt Count"
+	} // End of handling of the HISPANIC_CNT variable
+	
+	//Handles instances of the MALE_CNT variable
+	if `: list posof "male_cnt" in x' != 0 {
+	qui: rename male_cnt malecnt
+	qui: destring malecnt, replace ignore("*,-R %")
+	la var malecnt "Male Student Involvemnt Count"
+	} // End of handling of the MALE_CNT variable
+	
+	//Handles instances of the FEMALE_CNT variable
+	if `: list posof "female_cnt" in x' != 0 {
+	qui: rename female_cnt femalecnt
+	qui: destring femalecnt, replace ignore("*,-R %")
+	la var femalecnt "Female Student Involvemnt Count"
+	} // End of handling of the FEMALE_CNT variable
+	
+	//Handles instances of the TOTAL_STDNT_CNT variable
+	if `: list posof "total_stdnt_cnt" in x' != 0 {
+	qui: rename total_stdnt_cnt totstdcnt
+	qui: destring totstdcnt, replace ignore("*,-R %")
+	la var totstdcnt "Total Student Involvemnt Count"
+	} // End of handling of the TOTAL_STDNT_CNT variable
+	
+	//Handles instances of the TOTAL_UNIQUE_EVENT_CNT variable
+	if `: list posof "total_unique_event_cnt" in x' != 0 {
+	qui: rename total_unique_event_cnt totunqevntcnt
+	qui: destring totunqevntcnt, replace ignore("*,-R %")
+	la var totunqevntcnt "Total Unique Event Count"
+	} // End of handling of the TOTAL_UNIQUE_EVENT_CNT variable
+	
+	//Handles instances of the RPT_HEADER variable
+	if `: list posof "rpt_header" in x' != 0 {
+	qui: replace rpt_header=cond(`rx'(rpt_header, "Behavior Events", 1), "1", ///
+							  cond(rpt_header=="Behavior Events by Context", "2",        ///
+							  cond(rpt_header=="Behavior Events by Grade Level", "3",    ///
+							  cond(rpt_header=="Behavior Events by Locations"," 4",      ///
+							  cond(rpt_header=="Behavior Events by Socio-Economic Status", "5",  ///
+							  cond(rpt_header=="Discipline Resolutions", "6",            ///
+							  cond(rpt_header=="Discipline-Resolutions", "7",            ///
+							  cond(rpt_header=="Legal Sanctions", "8", ""))))))))        ///
+							  
+	qui: rename rpt_header rpthdr
+	qui: desting rpthdr,replace ignore("*,-R %")
+	la val rpthdr rpthdr
+	la var rpthdr Report Header
+	} // End of handling of the RPT_HEADER variable
+	
+	//Handles instances of the RPT_HEADER_ORDER variable
+	if `: list posof "rpt_header_order" in x' !=0 {
+	qui: rename rpt_header_order rpthdrodr
+	qui: destring rpthdrodr, replace ignore("*,-R %")
+	la var rpthdrodr "Report Header Order"
+	} // End of handling of the RPT_HEADER_ORDER variable
+	
+	//Handles instance of the RPT_LINE_ORDER variable
+	if `: list posof "rpt_line_order" in x' !=0 {
+	qui: rename rpt_line_order rptlnodr
+	qui: destring rptlnodr, replace ignore("*,-R %")
+	la var rptlnodr "Report Line Order"
+	} // End of handling of the RPT_LINE_ORDER variable
+	
+	//Handles instance of the SPENDING_PER_STDNT variable
+	if `: list posof "spending_per_stdnt" in x' !=0 {
```

Comment:
use `ppe` for this variable.  For the variable label use `"Per-Pupil Expenditure"`